src/graphql/queries.js

```diff
@@ -53,41 +53,41 @@ fragment WorkflowTreeDeltas on Deltas {
 fragment WorkflowTreeAddedData on Added {
   workflow {
     ...WorkflowData
-    cyclePoints: familyProxies(ids: ["root"], ghosts: true) {
+    cyclePoints: familyProxies (ids: ["root"], ghosts: true) {
       cyclePoint
     }
-    taskProxies(sort: { keys: ["cyclePoint"] }, ghosts: true) {
+    taskProxies (sort: { keys: ["name"], reverse: false }, ghosts: true) {
```

Comment:
That's why there was no order in the tasks. They were being sorted by cycle point (can't tell why, probably my bad :smile: ). Now sorting by name.